File: level68/homework/codwars11.py
#https://www.codewars.com/kata/5679aa472b8f57fb8c000047/train/python
def find_even_index(lst):
    left_sum = 0
    right_sum = sum(lst)
    for i, a in enumerate(lst):
        right_sum -= a
        if left_sum == right_sum:
            return i
        left_sum += a
    return -1

File: level56/homework/codwars11.js
//https://www.codewars.com/kata/526c7363236867513f0005ca/train/javascript
function isLeapYear(year) {
  if(0 == year%400) return true;
  if(0 == year%100) return false;
  if(0 == year%4) return true;
  return false;
}